### PR TITLE
fix: update timestamp util for calibrate input dataset

### PR DESCRIPTION
**Pull request by Jami159**
Update timestamp util

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/util.ts [hunk 1]
```diff
@@ -5,12 +5,26 @@ export const enum RunType {
...
 	let end = inputTimespan.end;
 	// If we have the min/max timestamp available from the csv asset use it
+	// Try to get `timestamp` first, then `t`
+	// TODO: figure out a more robust way to get the timestamp column from the csv asset
 	if (dataset) {
-		const tIndex = dataset.headers.indexOf('timestamp');
+		let tIndex = dataset.headers.indexOf('timestamp');
+		if (tIndex === -1) {
+			tIndex = dataset.headers.indexOf('t');
+		}
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/util.ts:27]: You should have and leverage a mappings data structure provided to you that tells you which column is the timestamp column.

File: packages/client/hmi-client/src/components/workflow/util.ts [hunk 2]
```diff
@@ -5,12 +5,26 @@ export const enum RunType {
...
 
-export const getTimespan = (inputTimespan: TimeSpan, dataset?: CsvAsset) => {
+export type DatasetType = {
+	data: { x: number; y: number }[];
+	label: string;
+	fill: boolean;
+	borderColor?: string;
+	borderWidth?: number;
+	borderDash?: number[];
+};
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/util.ts:15]: The naming does not reflect the purpose of this data structure. 

Probably more appropriate as `DataseriesConfig` or something?
Also consider moving the definition over to types/workflow.ts or types/SimulateConfig.ts

Comment (Jami159) [reply packages/client/hmi-client/src/components/workflow/util.ts]: name updated and moved to `SimulateConfig.ts`

File: packages/client/hmi-client/src/components/workflow/util.ts (hunk 1 above)
Comment (Jami159) [reply packages/client/hmi-client/src/components/workflow/util.ts:20]: Done!

**Pull request merged**